scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2637,6 +2778,14 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
     loop(Nil)
   }
 
+  def methodParamModifiers(): List[Mod] = {
+    if (isInlineSoftKw(token) && ahead(token.is[Ident])) {
```

Comment:
```suggestion
    if (isInlineSoftKw(token) && ahead(token.is[Ident] || DclIntro.unapply(token) || Modifier.unapply(token))) {
```
?

Reply:
you wanted to add this comment to `inlineDefOrOpaque` piece of code not `methodParamModifiers`, right?

Reply:
Hmm... yeah this might not be an issue. It might make sense to have `Modifier.unapply(token)` if at any point we have`inline other_modifier param`, but currently we might just leave it as is.